app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+end
+
+# A method that returns the 3 most recent posts for a given user.
+def recent_3_posts
+  post.last(3)
+end
```

Comment:
- [ ] This `recent_3_posts` method should be inside `User` class

Reply:
Done.